### ISSUE TITLE
Selecting folder

### ISSUE THREAD

**Issue by theRAGEhero** [antiboredom/videogrep#116]
Hello Sam,

am I doing something wrong, or at the moment is not possible to select a folder instead of a video? I have many .mp4 files with their .srt subtitle file and I would like to do a supercut from all those files.

PS. I've also emailed you, but probably it ended up in spam.

Thank you

**Comment by marcospozzo** [comment]
Hi! Have you tried with *.mp4?

**Comment by theRAGEhero** [comment]
Hi! Thank you, but unfortunately it doesn't work.

**Comment by antiboredom** [comment]
@theRAGEhero can you paste in the exact command you used and also the contents of the folder?

**Comment by theRAGEhero** [comment]
Sorry for my late reply.

This is the command. Inside the videogrep folder I have two .mp4 files with their .srt 
I can videocut them singularly, but not as a folder.

```
"./videogrep/*.mp4" --search 'legge' -n 1
No subtitle file found for  ./videogrep/*.mp4
```

I'm just looking at some old videcut that I did and apparently I already did it with multiple videos, still I don't remember if I did it with a folder or inserting manually every video source.

Thank you.
